.gitignore

```diff
@@ -24,6 +24,7 @@ pickle-email-*.html
 config/initializers/secret_token.rb
 config/secrets.yml
 config/application.yml
+.env
```

Comment:
what about using [rails credentials?](https://kirillshevch.medium.com/encrypted-secrets-credentials-in-rails-6-rails-5-1-5-2-f470accd62fc)